src/Symfony/Bundle/SecurityBundle/Tests/DataCollector/SecurityDataCollectorTest.php

```diff
@@ -115,7 +115,7 @@ public function testCollectSwitchUserToken()
         $this->assertSame('yceruto', $collector->getImpersonatorUser());
         $this->assertSame(SwitchUserToken::class, $collector->getTokenClass()->getValue());
         $this->assertTrue($collector->supportsRoleHierarchy());
-        $this->assertSame(['ROLE_USER', 'ROLE_PREVIOUS_ADMIN'], $collector->getRoles()->getValue(true));
+        $this->assertSame(['ROLE_USER', 'ROLE_KOALA'], $collector->getRoles()->getValue(true));
```

Comment:
One role would be enough, wouldn't it?

Reply:
Don't you like koala 😭 

haha, ok, I'll remove it